SwiftWisdom/Core/UIKit/UIView+SafeArea.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  UIView+SafeArea.swift
+//  SwiftWisdom
+//
+//  Created by Persian, Alex on 11/3/17.
+//  Copyright © 2017 Intrepid. All rights reserved.
+//
+
```

Comment:
Maybe just throw this link in here at the top?
https://developer.apple.com/documentation/uikit/uiview/2891102-safearealayoutguide